kubectl-plugin/test/e2e/kubectl_ray_test.go

```diff
@@ -0,0 +1,38 @@
+package e2e
+
+import (
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestKubectlRayGet(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Kubectl Ray Get")
+}
+
+var _ = Describe("Calling ray plugin `get` command", func() {
+	It("should succeed without error", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).NotTo(HaveOccurred(), "%s: %s", err, output)
+	})
+
+	It("should not succeed", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get", "fakeclustername", "anotherfakeclustername")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).To(HaveOccurred())
+		Expect(output).ToNot(ContainElements("fakeclustername"))
+	})
+
+	It("will have a `namespace` and should succeed  without error", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get", "--namespace", "default")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).NotTo(HaveOccurred(), "%s: %s", err, output)
```

Comment:
Can we create a cluster and validate the output contains the cluster name too?

Reply:
I changed the first test to succeed but also check if the output contains the ray cluster. Ray cluster gets created during the github workflow.